examples/example_taggged_signals.rs

```diff
@@ -46,9 +46,9 @@ impl actix::StreamHandler<WindowSignal> for WindowActor {
             WindowSignal::ClickButton => {
                 let addend_id = self.next_addend_id;
                 self.next_addend_id += 1;
-                let widgets = self.factories.row_addend.build()
-                    .connect_tagged_builder_signals(ctx, addend_id)
-                    .widgets().unwrap();
+                let mut builder = self.factories.row_addend.instantiate();
+                builder.connect_signals_tagged(addend_id, ctx);
```

Comment:
I guess Rust can deduce that this does not need `::<AddendSignal>`, but it's probably a good idea to add it anyway...